beekeeper-cleanup/pom.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
     <artifactId>beekeeper-parent</artifactId>
     <groupId>com.expediagroup</groupId>
-    <version>1.1.5-SNAPSHOT</version>
+    <version>1.1.3-SNAPSHOT</version>
```

Comment:
Shouldn't this be 1.1.4-SNAPSHOT as the 1.1.4 release is what failed? You also need to delete the tag (https://github.com/ExpediaGroup/beekeeper/tags) and revert the change to the changelog.

Reply:
Yeah I had this in my head as a 1.1.4 release, should have been 1.1.3. Will update this in the changelog too.